src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -209,6 +212,7 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
       })
       .subscribe((feedbackSession: FeedbackSession) => {
         this.sessionEditFormModel = this.getSessionEditFormModel(feedbackSession);
+        this.feedbackSessionModelBeforeEditing = this.sessionEditFormModel;
```

Comment:
I don't think this will work as the two variable refer to the same object.

```suggestion
        this.feedbackSessionModelBeforeEditing = this.getSessionEditFormModel(feedbackSession);
```